database/migrations/2021_11_22_093555_migrate_thread_feed_to_timestamp.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use App\Models\Thread;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class MigrateThreadFeedToTimestamp extends Migration
+{
+    public function up()
+    {
+        Schema::table('threads', function (Blueprint $table) {
+            $table->timestamp('last_active_at')
```

Comment:
Let's rename this to `last_reply_at`? That makes it more clear what this represents. Also rename related methods, etc.

Reply:
The reason I called it last_active_at is the time stamp will first be set when the thread is created. If nobody replies, this will never update

Reply:
Cool, that makes sense. Then let's name this: `last_activity_at`.